src/components/comments.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { fetchAllCommentsByArticleId } from "../api";
+
+export const Comments = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    fetchAllCommentsByArticleId(article_id).then((data) => {
+      setComments(data.comments);
+    });
+  }, [article_id]);
+
+  return (
+    <div className="comments">
+      <h3 className="comments-header">Comments</h3>
+
+      {comments.map((comment) => {
+        return (
```

Comment:
Make this a separate <Comment /> component for ultimate reusability please :)

Reply:
Thanks for the feedback.. I will make those changes :)